src/kernels/utf8.rs

```diff
@@ -1,4 +1,4 @@
-use arrow2::array::Utf8Array;
+use arrow2::array::{BooleanArray, Utf8Array};
 
```

Comment:
we are planning on depreciating src/kernels, I would just put the kernels in the same file with DataArray function.